### PR TITLE
Quantize vit_b_16 tutorial - Part 1

### PR DESCRIPTION
**Pull request by cpuhrsch**

### CODE REVIEW

File: tutorials/quantize_vit/bfloat16_code [hunk 1]
```diff
@@ -0,0 +1,1351 @@
...
+V0315 17:55:08.077000 140041589225280 torch/_inductor/graph.py:1258] [0/0] [__output_code]     return print_performance(fn, times=times, repeat=repeat)
+V0315 17:55:08.077000 140041589225280 torch/_inductor/graph.py:1258] [0/0] [__output_code] 
+V0315 17:55:08.077000 140041589225280 torch/_inductor/graph.py:1258] [0/0] [__output_code] 
+V0315 17:55:08.077000 140041589225280 torch/_inductor/graph.py:1258] [0/0] [__output_code] if __name__ == "__main__":
+V0315 17:55:08.077000 140041589225280 torch/_inductor/graph.py:1258] [0/0] [__output_code]     from torch._inductor.wrapper_benchmark import compiled_module_main
+V0315 17:55:08.077000 140041589225280 torch/_inductor/graph.py:1258] [0/0] [__output_code]     compiled_module_main('None', benchmark_compiled_module)
+V0315 17:55:08.077000 140041589225280 torch/_inductor/graph.py:1258] [0/0] [__output_code] 
+I0315 17:55:08.079000 140041589225280 torch/_inductor/graph.py:1264] [0/0] [__output_code] Output code written to: /tmp/torchinductor_cpuhrsch/2i/c2ixftylrwvvc3swfutdqklg6xb2w47xlwmfdmtgktp4yb4kzkro.py
```
Comment (msaroufim) [inline tutorials/quantize_vit/bfloat16_code:1351]: Maybe check in this file instead?
/tmp/torchinductor_cpuhrsch/2i/c2ixftylrwvvc3swfutdqklg6xb2w47xlwmfdmtgktp4yb4kzkro.py

File: tutorials/quantize_vit/run.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Run bfloat16 version
+TORCH_LOGS='graph_breaks,recompiles' python run_vit_b.py
+
+# Run dynamic quantized version
+TORCH_LOGS='graph_breaks,recompiles' python run_vit_b_quant.py
+
+# Store the output code for further inspection
+TORCH_LOGS='output_code' python run_vit_b.py 2> bfloat16_code
+TORCH_LOGS='output_code' python run_vit_b_quant.py 2> quant_code
```
Comment (msaroufim) [inline tutorials/quantize_vit/run.sh:11]: I think what people might expect to see here is the fused quant and dequant in the generated code so we just add some comments to the checked in code so people can inspect

File: tutorials/quantize_vit/run_vit_b_quant.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+# Set the model to evaluation mode
+model.eval().cuda().to(torch.bfloat16)
+
+# Input tensor (batch_size, channels, height, width)
+input_tensor = torch.randn(1, 3, 224, 224, dtype=torch.bfloat16, device='cuda')
+
+## Quantization code - start
+from torchao.quantization import quant_api
+quant_api.change_linear_weights_to_int8_dqtensors(model)
```
Comment (jerryzh168) [inline tutorials/quantize_vit/run_vit_b_quant.py:15]: @cpuhrsch not sure if we discussed about this, do you have any thoughts/comments on putting these under the unified quantization API (https://github.com/pytorch-labs/ao/blob/main/torchao/quantization/quant_api.py#L52)?

File: tutorials/quantize_vit/bfloat16_code (hunk 1 above)
Comment (cpuhrsch) [reply tutorials/quantize_vit/bfloat16_code]: Yes, I can make it a `.py` file. I think the name isn't great. torch should have a way of outputting these in a neat location more officially.

File: tutorials/quantize_vit/run.sh (hunk 2 above)
Comment (cpuhrsch) [reply tutorials/quantize_vit/run.sh]: I'll try to add more pictures and also isolate the kernel. It's much clearer from the traces.

**Pull request merged**